workspace_hack/Cargo.toml

```diff
@@ -20,7 +20,9 @@ clap = { version = "4", features = ["derive", "string"] }
 crossbeam-utils = { version = "0.8" }
 either = { version = "1" }
 fail = { version = "0.5", default-features = false, features = ["failpoints"] }
+futures = { version = "0.3" }
 futures-channel = { version = "0.3", features = ["sink"] }
+futures-executor = { version = "0.3" }
```

Comment:
Why is this required? Cannot see it being used..

Reply:
Hmm, `cargo hakari generate` wants to put it there..

To see where it's used, I temporarily removed it from workspace_hack, and ran `cargo tree`. It's used in two places now:

* (several packages) -> futures -> futures-executor
* comute_tools -> opentelemetry -> opentelemetry_sdk -> futures-executor

The second dependency to it is new, so I guess "hakari" wants to add it to workspace_hack now because there are now two dependency paths to it.

Reply:
You are correct. Great idea disabling the workspace hack and then querying. Very odd to see it being depended on directly: https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-sdk/Cargo.toml#L21 and used directly from https://github.com/open-telemetry/opentelemetry-rust/blob/0ef87ad78a6b83a91c14a7926efae101a8b111c2/opentelemetry-sdk/src/metrics/controllers/basic.rs#L174. Maybe there's a good reason.

Resolving...